loom/script/runtime/lsLuaState.cpp

```diff
@@ -86,8 +85,8 @@ void LSLuaState::open()
 {
     assert(!L);
 
-    L = lua_newstate(lsLuaAlloc, NULL);
-    //L = luaL_newstate();
+    //L = lua_newstate(lsLuaAlloc, NULL);
+    L = luaL_newstate();
```

Comment:
Oh also, from what I've read, the internal lua allocator is supposedly very optimized and tuned to lua, so it might be faster than using the global/default/jemalloc one, but I haven't tested this myself.